_data/affiliations.yml

```diff
@@ -1,6 +1,5 @@
 - name: The University of Manchester
   image_url: /images/institutions/logo-university-of-manchester.svg
-  pid: https://ror.org/02aswte26
```

Comment:
Can we replace the ror with the right one - https://ror.org/027m9bs27

Reply:
Fixed via [946e0d1](https://github.com/EVERSE-ResearchSoftware/RSQKit/commit/946e0d16637518655e8bbecc84daae1aa1ec1928)
 ·